### DISCUSSION TITLE
Is Flux refiner training possible?

### DISCUSSION THREAD

**Discussion by matabear-wyx** [bghira/SimpleTuner#1073] [General]
Maybe some two-stage training like SDXL?

**Comment by bghira** [comment]
it is continuous time schedule so not really. flow-matching in general is a disappointment for these reasons

**Reply by matabear-wyx** [reply]
yet for a continuous time scheduler, the concept of a 'refiner' still holds, right?
Since there are no explicit discrete steps, there can be a seamless transition between the base generation and refiner stages. I think it needs to be redesigned in terms of dividing the time intervals and adjusting the noise reduction strategies.
But I don't really have any idea on how to design this for now :(
